README.org

```diff
@@ -116,16 +116,16 @@
   - Proton
   - Lepton
 
-** F&A
+** FAQ
 
   - *Black pixels around the selected tab bottom corners* \\
     [[https://user-images.githubusercontent.com/5571586/120401980-edf58a00-c2f5-11eb-9e64-ce50c5b189b2.png]]
 
-  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]]. \\
-  Or =about:config='s =svg.context-properties.content.enabled= to =true=
+  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]], \\
+  or set =about:config='s =svg.context-properties.content.enabled= to =true= .
 
   - *Some panel menu items doesn't show* \\
     [[https://user-images.githubusercontent.com/25581533/120487528-93b40200-c3a5-11eb-98ad-3498beb9f38e.png]]
 
-  Please follow [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]]. \\
-  Or =icons= directory copy.
+  Please follow the [[https://github.com/black7375/Firefox-UI-Fix/wiki/Installation-Guide][Installation Guide]], \\
```

Comment:
Would it be better to have `the`?

Reply:
thats a typo.. The commit is an addition of ‘on’.

Reply:
What do you mean?